cocos/ui/UIWidget.h

```diff
@@ -956,6 +958,7 @@ class CC_GUI_DLL Widget : public ProtectedNode, public LayoutParameterProtocol
     Vec2 _positionPercent;
 
     bool _hitted;
+    const Camera *_hittedByCamera;
```

Comment:
// weak reference of the camera which made the widget passed the hit test when response touch begin event
    // it's useful in the next touch move/end events
    const Camera *_hittedByCamera;